controllers/servicebinding_controller.go

```diff
@@ -760,6 +759,16 @@ func (r *ServiceBindingReconciler) addInstanceInfo(ctx context.Context, binding
 	return metadata, nil
 }
 
+func getInstanceName(instance *servicesv1.ServiceInstance) []byte {
```

Comment:
getInstanceNameForSecretCredentials
(or something like that)

Reply:
done